aliasmgr/aliasmgr.go

```diff
@@ -53,10 +53,10 @@ var (
 	// endBlockHeight is the ending block height of the alias range.
 	endBlockHeight = 16_250_000
 
-	// startingAlias is the first alias ShortChannelID that will get
+	// StartingAlias is the first alias ShortChannelID that will get
```

Comment:
I was checking the Bolts but I do not see the `startingBlockHeight`. 

Is this standardized or each implementation will set their own start block?  I expected to find it in [BOLT2](https://github.com/lightning/bolts/blob/b38156b9510c0562cf50f8758a64602cc0315c19/02-peer-protocol.md)

Reply:
We define this in the `aliasmgr` package - it's implementation-defined